pkg/broker/handler/processors/deliver/processor.go

```diff
@@ -170,6 +170,13 @@ func (p *Processor) deliver(ctx context.Context, target *config.Target, broker *
 	if err != nil {
 		logging.FromContext(ctx).Error("failed to add status code tags to context", zap.Error(err))
 	}
+
+	// Insert subscriber info into context.
+	cctx, err = metrics.AddSubscriberTags(ctx, target)
```

Comment:
```suggestion
	cctx, err = metrics.AddSubscriberTags(cctx, target)
```

Reply:
Do we care about the timeout case above ? 
Maybe we should move the adding of the subscriber tag higher up to before we call sendMsg().